build/release.mk

```
@@ -35,10 +35,10 @@ release/sign:
 	@echo "=== $(INTEGRATION) === [release/sign] signing packages"
 	@bash $(CURDIR)/build/sign.sh
 
-
 .PHONY : release/publish
 release/publish:
 	@echo "=== $(INTEGRATION) === [release/publish] publishing artifacts"
+	@bash $(CURDIR)/build/upload_packages_gh.sh
```

Might be better way of calling the script:
```suggestion
	@$(shell $(CURDIR)/build/upload_packages_gh.sh)
```

---

Or just:
```suggestion
	$(CURDIR)/build/upload_packages_gh.sh
```